File: 프로그래머스/2016년.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(int a, int b) {
	string answer[] = { "THU","FRI","SAT","SUN","MON","TUE","WED" };
	int month[] = { 31,29,31,30,31,30,31,31,30,31,30,31 };
	int sum = 0;

	for (int i = 0; i < a - 1; i++) {
		sum += month[i];
	}
	sum += b;
	return answer[sum % 7];
}

int main() {

	cout << solution(5, 24) << endl;
}

File: 프로그래머스/2016년.py
def solution(a, b):
    answer = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    sum = b-1
    for i in range(a - 1):
        sum += month[i]
    return answer[sum % 7]


print(solution(5, 24))